### ISSUE TITLE
Feature Request: Use field label in export columns instead of handle.

### ISSUE THREAD

**Issue by mark-chief** [verbb/formie#928]
Would be great to be able to use the Field label in the export columns instead of the handle. 

Sometimes when a field is re-used across multiple forms the handle may stay the same but the label will change. Meaning when admin export the data, the column headings dont match the field labels. 

Thanks! :)

**Comment by engram-design** [comment]
Added for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.1",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Added in [2.0.2](https://github.com/verbb/formie/releases/tag/2.0.2)

**Comment by 0xElectric** [comment]
Is there any fixes for formie version "1.6.26" for craft 3?
